kumo-core/src/main/java/com/kennycason/kumo/bg/PixelBoundaryBackground.java

```diff
@@ -52,6 +64,30 @@ public PixelBoundaryBackground(final File file) throws IOException {
     public PixelBoundaryBackground(final String filepath) throws IOException {
         this(new File(filepath));
     }
+
+    private BufferedImage TransBackground(BufferedImage srcImage, int alpha) throws Exception {
+        int imgHeight = srcImage.getHeight();
+        int imgWidth = srcImage.getWidth();
+        //get the pixel of left upper corner
+        int c = srcImage.getRGB(3, 3);
```

Comment:
I think we should probably pass in an `int backgroundColor` to the function instead of relying on pixel 3x3 which could be anything.